src/parser/core/modules/Overheal.tsx

```diff
@@ -235,6 +251,17 @@ export class Overheal extends Analyser {
 
 		const guid = event.cause.type === 'action' ? event.cause.action : event.cause.status
 		const name = event.cause.type === 'action' ? this.data.getAction(guid)?.name : this.data.getStatus(guid)?.name
+
+		const bucketId = this.overrideHealBucket(event, petHeal)
+		if (bucketId >= 0) {
+			for (const trackedHeal of this.trackedOverheals) {
+				if (trackedHeal.bucketId === bucketId) {
+					this.debug(`Heal ${name} (${guid}) at ${event.timestamp} MANUALLY shoved into bucket ${trackedHeal.name.props.default}`)
```

Comment:
small nit: it should be ${trackedHeal.name.props.defaults} (see the normal 'matched into category' debug string)

Reply:
The worst part about this? I'm the one who wrote the other debug string, so I should know this.